tools/dynamo/gb_id_mapping.py

```diff
@@ -0,0 +1,146 @@
+
+
+def clean_string(s):
+    if s is None:
+        return None
+    if isinstance(s, str):
+        s = re.sub(r'^f["\']', r'"', s)
+        s = re.sub(r'["\'] f["\']', " ", s)
+        s = re.sub(r'^["\'](.*)["\']$', r"\1", s)
+        s = re.sub(r"\s*\n\s*", " ", s)
+        s = s.replace('\\"', '"').replace("\\'", "'")
+        s = s.replace("\\", "")
+        s = re.sub(r'" "', " ", s)
+        return s
+    return s
+
+
+def expand_hints(hints):
+    from torch._dynamo import graph_break_hints
+
+    expanded_hints = []
+    for hint in hints:
+        if isinstance(hint, str):
+            if "*graph_break_hints.USER_ERROR" or "graph_break_hints.USER_ERROR" in hint:
+                expanded_hints.extend(graph_break_hints.USER_ERROR)
+            elif "*graph_break_hints.DYNAMO_BUG" or "graph_break_hints.DYNAMO_BUG" in hint:
+                expanded_hints.extend(graph_break_hints.DYNAMO_BUG)
+            elif "*graph_break_hints.DIFFICULT" or "graph_break_hints.DIFFICULT" in hint:
+                expanded_hints.extend(graph_break_hints.DIFFICULT)
+            elif "*graph_break_hints.FUNDAMENTAL" or "graph_break_hints.FUNDAMENTAL" in hint:
+                expanded_hints.extend(graph_break_hints.FUNDAMENTAL)
+            elif "*graph_break_hints.SUPPORTABLE" or "graph_break_hints.SUPPORTABLE" in hint:
+                expanded_hints.extend(graph_break_hints.SUPPORTABLE)
+            elif "*graph_break_hints.CAUSED_BY_EARLIER_GRAPH_BREAK" or "graph_break_hints.CAUSED_BY_EARLIER_GRAPH_BREAK" in hint:
+                expanded_hints.extend(graph_break_hints.CAUSED_BY_EARLIER_GRAPH_BREAK)
+            elif "*graph_break_hints.INFERENCE_MODE" or "graph_break_hints.INFERENCE_MODE" in hint:
+                expanded_hints.extend(graph_break_hints.INFERENCE_MODE)
+            else:
+                expanded_hints.append(hint)
+        else:
+            expanded_hints.append(hint)
+
+    return expanded_hints
+
+
+def find_unimplemented_v2_calls():
+    dynamo_dir = Path(__file__).parent.parent.parent / "torch" / "_dynamo"
+    results = []
+
+    for file_path in dynamo_dir.glob("**/*.py"):
+        with open(file_path, "r") as f:
+            source = f.read()
+            try:
+                tree = ast.parse(source)
+
+                for node in ast.walk(tree):
+                    if isinstance(node, ast.Call) and hasattr(node, "func"):
+                        is_unimplemented = False
+                        if (
+                            isinstance(node.func, ast.Name)
+                            and node.func.id == "unimplemented_v2"
+                        ):
+                            is_unimplemented = True
+                        elif (
+                            isinstance(node.func, ast.Attribute)
+                            and node.func.attr == "unimplemented_v2"
+                        ):
+                            is_unimplemented = True
+
+                        if is_unimplemented:
```

Comment:
`node.func` could be arbitrary, e.g. consider `a["fn"](x)` - func will be a `ast.Subscript`. Let's keep things simple and just look for `ast.Name`s with `unimplemented_v2`.